### PR TITLE
Added spelled-out aliases for abbreviated predicate cases.

### PR DESCRIPTION
**Pull request by harlanhaskins**

### CODE REVIEW

File: Sources/LLVM/Predicates+ExpandedNames.swift [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    static let orderedGreaterThan = ogt
+    /// Ordered greater than or equal.
+    static let orderedGreaterThanOrEqual = oge
+    /// Ordered less than.
+    static let orderedLessThan = olt
+    /// Ordered less than or equal.
+    static let orderedLessThanOrEqual = ole
+    /// Ordered and not equal.
+    static let orderedNotEqual = one
+    /// Oredered (no nans).
```
Comment (CodaFi) [inline Sources/LLVM/Predicates+ExpandedNames.swift]: Can you fix this and the corresponding one in `RealPredicate`?

**Review by CodaFi — CHANGES_REQUESTED** [review]
I think these should be the real cases and the aliases should be reversed to migrate users to the new spelled-out cases.

**Comment by harlanhaskins** [comment]
I've been waffling back and forth, but I agree with you. Should we bother with deprecation warnings or just rip the band-aid off and make an @channel on the Slack?

**Comment by CodaFi** [comment]
I think a single point-release containing a set of migration aliases will be fine.  I doubt anybody is using `fcmp` heavily enough at this point that it will be a problem.

File: Sources/LLVM/IRBuilder.swift [hunk 2]
```diff
@@ -68,42 +70,44 @@ public enum RealPredicate {
...
   /// Ordered less than.
-  case olt
+  case orderedLessThan
   /// Ordered less than or equal.
-  case ole
+  case orderedLessThanOrEqual
   /// Ordered and not equal.
-  case one
+  case orderedNotEqual
   /// Oredered (no nans).
```
Comment (CodaFi) [inline Sources/LLVM/IRBuilder.swift]: Documentation flub still exists.

**Review by CodaFi — APPROVED** [review]

**Comment by harlanhaskins** [comment]
Will merge once CI passes

**Comment by harlanhaskins** [comment]
🚢 :it:

**Pull request merged**